website/docs/docs/core/pip-install.md

```diff
@@ -154,3 +154,107 @@ python -m pip install \
 ```
 
 Or, better yet, just install the package(s) you need!
+
+<VersionBlock firstVersion="1.8">
+
+### Install prereleases of dbt-adapters
+
+A prerelease adapter is a version released before the final, stable version. It allows users to test new features, provide feedback, and get early access to upcoming functionality &mdash; ensuring your system will be ready for the final release.
+
+Using a prerelease of an adapter has many benefits such as granting you early access to new features and improvements ahead of the stable release. As well as compatibility testing, allowing you to test the adapter in your environment to catch integration issues early, ensuring your system will be ready for the final release. 
+
+Note that using a prerelease version before the final, stable version means the version isn't fully optimized and can result in unexpected behavior. Additionally, frequent updates and patches during the prerelease phase may require extra time and effort to maintain.
+
+To install prerelease versions of dbt Core and your adapter, use this command (replace `dbt-adapter-name` with your adapter)
+
+```shell
+python3 -m pip install --pre dbt-core dbt-adapter-name
+````
+
+For example, if you’re using Snowflake, you would use the command:
+
+
+```shell
+python3 -m pip install --pre dbt-core dbt-snowflake
+````
+
+We recommend you install prereleases in a [virtual Python environment](https://packaging.python.org/en/latest/guides/installing-using-pip-and-virtual-environments/). For example, to install a prerelease in a `POSIX bash`/`zsh` virtual Python environment, use the following commands:
+
+```shell
+dbt --version
+python3 -m venv venv
+source venv/bin/activate
+python3 -m pip install --upgrade pip
+python3 -m pip install --pre dbt-core dbt-adapter-name
+source venv/bin/activate
+dbt --version
+```
+Note, this will also install any pre-releases of all dependencies.
+
+#### Activate your virtual environment 
+
+To install or use packages within your virtual environment:
+
+- Activate the virtual environment to add its specific Python and `pip` executables to your shell’s PATH. This ensures you use the environment’s isolated setup. 
+
+For more information, refer to [Create and use virtual environments](https://packaging.python.org/en/latest/guides/installing-using-pip-and-virtual-environments/#create-and-use-virtual-environments).
+
+Select your operating system and run the following command to activate it:
+
+<Expandable alt_header="Unix/macOS" >
+
+1. Activate your virtual environment: 
+
+```shell
+source .venv/bin/activate
+which python
+venv/bin/python
+  
+  ```
+  2. Install the prerelease using the following command:
+
+
+```shell
+python3 -m pip install --pre dbt-core dbt-adapter-name
+source .venv/bin/activate
+dbt --version
+```
+
+</Expandable>
```

Comment:
We don't have any pre-releases that are higher than the most recent release, so I can't test if these commands work as intended or not. But I don't see anything wrong with them. We need to wait a few months for another pre-release to test it out.

In the meantime, choosing a specific pre-release version from the [PyPI release history](https://pypi.org/project/dbt-core/#history) worked for me:

```python
python3 -m pip install "dbt-core==1.9.0rc2" "dbt-snowflake==1.9.0rc1"
```